src/table/table.scss

```
@@ -400,7 +400,7 @@
     align-items: center;
     justify-content: center;
 
-    > svg {
+    > .iui-icon {
```

There are already meaningful icon classes above (.iui-cell-end-icon, .iui-cell-start-icon)

```suggestion
    svg {
```